tests/Serializer/AbstractSerializerTest.php

```diff
@@ -65,6 +65,41 @@ public function testObjectsAreNotStrings(): void
         $this->assertSame(['key' => 'value'], $result);
     }
 
+    public function traversableDataProvider(): \Generator
+    {
+        $getArray = static function (): array {
+            return [
+                'a' => 'foo',
```

Comment:
I think that for this test it's enough to use a simple list instead of an hashmap

Reply:
You're right (and that'll make it even easier to inline) :+1: